src/EditorFeatures/CSharpTest/PdbSourceDocument/PdbSourceDocumentTests.cs

```diff
@@ -792,6 +867,8 @@ private static void CompileTestSource(string path, string source, Project projec
                 emitOptions = emitOptions.WithDebugInformationFormat(DebugInformationFormat.Pdb);
             }
 
+            emitOptions = emitOptions.WithDefaultSourceFileEncoding(source.Encoding);
```

Comment:
Do we have any tests with fallback encoding?

Reply:
No good point.